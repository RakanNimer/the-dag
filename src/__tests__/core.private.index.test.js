import All from '../core/private';

describe('Private methods', () => {
  test('export correctly', () => {
    expect(All).toMatchSnapshot();
  });
});
