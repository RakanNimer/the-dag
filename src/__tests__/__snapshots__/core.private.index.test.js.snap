// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Private methods export correctly 1`] = `
Object {
  "privateMethods": Object {
    "addPossibleSources": [Function],
    "addPossibleTargets": [Function],
    "createEdge": [Function],
    "createNode": [Function],
    "generateEdgeIDFromSourceAndTargetIDs": [Function],
  },
}
`;
